src/gsi/m_berror_stats.f90

```diff
@@ -725,7 +950,7 @@ subroutine sethwlloz_(hwlloz,mype)
    do k=1,nnnn1o
       k1=levs_id(k)
       if ( k1>0 ) then
-      write(6,*) myname_,'(PREWGT): mype = ',mype, k1
+      if(mype==0) write(6,*) myname_,'(PREWGT): mype = ',mype, k1
```

Comment:
if it's only for PE0, do we need `mype = ', mype`?

Reply:
Three write statements in `sethwlloz_` have the same structure - write limited to mpi rank 0 and prints mpi rank 0 to stdout.  By including the mpi rank in printout the user know the task from which the printout originated without needing to refer to the code.  Is this necessary?  No.  One can turn on task tracing via environment variables or command line arguments.  Remove `mype` from this and similar lines in `m_berror_stats.f90`.  Done at 68d52ac.